runtime/parachains/src/session_info.rs

```diff
@@ -38,6 +38,8 @@ pub trait Config:
 
 decl_storage! {
 	trait Store for Module<T: Config> as ParaSessionInfo {
+		/// Assignment keys for the current session.
+		AssignmentKeys: Vec<AssignmentId>;
```

Comment:
would prefer a slightly different name that indicates it can be 'off-by-one' at session boundaries.

Reply:
I would appreciate suggestions. But the `inclusion` module for example, store the validator ids with the same naming scheme:
```rust
		/// The current validators, by their parachain session keys.
		Validators get(fn validators) config(validators): Vec<ValidatorId>;
```

Reply:
Let me know what you think about c70ee27.

Reply:
That's fine, thanks